row.py

```diff
@@ -795,6 +796,143 @@ def download_run(bucket, run_name):
     return location.joinpath(run_name)
 
 
+def download_ocr_results(bucket_name, run_name, out_dir):
+    """download ocr results from a GCP bucket
+
+    Args:
+        bucket (str): the name of the bucket
+        run_name (str): the name of the run to get files from
+        out_dir (str): where to save the results
+
+    Returns:
+        str: the location of the files
+    """
+    if bucket_name.startswith("gs://"):
+        bucket_name = bucket_name[5:]
+
+    if "STORAGE_CLIENT" not in globals():
+        STORAGE_CLIENT = google.cloud.storage.Client()
```

Comment:
```suggestion
```

**todo**: remove this as it circumvents the reasons when the client is created. If you need the client locally, then satisfy the environment variables which allow it to be created.